Sources/SwiftInspectorVisitors/Tests/FileVisitorSpec.swift

```diff
@@ -218,6 +246,14 @@ final class FileVisitorSpec: QuickSpec {
           expect(matching.count) == 1
         }
 
+        it("finds Array.InnerTypealias") {
```

Comment:
Is this test file an "integration" test of sorts. I assume file visitor delegates to a lot of other sub visitors?

Reply:
Yes, this file is doing integration tests since it's delegating to sub visitors.